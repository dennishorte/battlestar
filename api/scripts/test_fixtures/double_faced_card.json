{
  "id": "card-id-456",
  "lang": "en",
  "layout": "transform",
  "card_faces": [
    {
      "name": "Delver of Secrets",
      "mana_cost": "{U}",
      "type_line": "Creature — Human Wizard",
      "oracle_text": "At the beginning of your upkeep, look at the top card of your library. You may reveal that card. If an instant or sorcery card is revealed this way, transform Delver of Secrets.",
      "power": "1",
      "toughness": "1",
      "colors": ["U"],
      "image_uris": {
        "art_crop": "https://example.com/delver_front.jpg",
        "normal": "https://example.com/delver_front_normal.jpg"
      }
    },
    {
      "name": "Insectile Aberration",
      "type_line": "Creature — Human Insect",
      "oracle_text": "Flying",
      "power": "3",
      "toughness": "2",
      "colors": ["U"],
      "image_uris": {
        "art_crop": "https://example.com/delver_back.jpg",
        "normal": "https://example.com/delver_back_normal.jpg"
      }
    }
  ],
  "color_identity": ["U"],
  "legalities": {
    "standard": "not_legal",
    "modern": "legal",
    "legacy": "legal",
    "vintage": "legal",
    "commander": "legal"
  },
  "cmc": 1,
  "rarity": "uncommon",
  "set": "TST",
  "collector_number": "51"
} 