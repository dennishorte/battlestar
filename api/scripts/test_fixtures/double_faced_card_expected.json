{
  "data": {
    "id": "card-id-456",
    "card_faces": [
      {
        "name": "Delver of Secrets",
        "mana_cost": "{U}",
        "type_line": "Creature — Human Wizard",
        "oracle_text": "At the beginning of your upkeep, look at the top card of your library. You may reveal that card. If an instant or sorcery card is revealed this way, transform Delver of Secrets.",
        "power": "1",
        "toughness": "1",
        "colors": ["U"],
        "image_uri": "https://example.com/delver_front.jpg"
      },
      {
        "name": "Insectile Aberration",
        "mana_cost": "{U}",
        "type_line": "Creature — Human Insect",
        "oracle_text": "Flying",
        "power": "3",
        "toughness": "2",
        "colors": ["U"],
        "image_uri": "https://example.com/delver_back.jpg"
      }
    ],
    "collector_number": "51",
    "color_identity": ["U"],
    "legal": ["modern", "legacy", "vintage", "commander"],
    "layout": "transform",
    "rarity": "uncommon",
    "set": "TST",
    "cmc": 1,
    "digital": null,
    "name": "Delver of Secrets // Insectile Aberration",
    "colors": ["U"],
    "type_line": "Creature — Human Wizard // Creature — Human Insect"
  },
  "source": "scryfall"
} 